manager/state/raft/membership/cluster.go

```diff
@@ -229,8 +249,7 @@ func (c *Cluster) CanRemoveMember(from uint64, id uint64) bool {
 			continue
 		}
 
-		connState, err := m.Conn.State()
-		if err == nil && connState == grpc.Ready {
+		if err := m.HealthCheck(); err == nil {
```

Comment:
Yeah, it can :/ I dunno how to check aliveness faster, though. I'll think about it.